co-log-benchmark/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2018 Alexander Vershilov
```

Comment:
I wonder, how should I update the LICENSE if I want to do something with the benchmarks (like, refactor or add new ones)? :thinking:

Reply:
oh.. that was autogenerated.